docs/iris/example_code/Meteorology/inset_plot.py

```diff
@@ -0,0 +1,73 @@
+"""
+Test Data Showing Inset Plots
+=============================
+
+This example demonstrates the use of a single 3D data cube with time, latitude
+and longitude dimensions to plot a latitude-averaged temperature series with
+an inset plot of the data region.
+
+"""
+
+###############################################################################
+
+import matplotlib.pyplot as plt
+import numpy as np
+import iris
+import cartopy.crs as ccrs
+import iris.quickplot as qplt
+import iris.plot as iplt
+
+###############################################################################
+
+
+def main():
+    iris.FUTURE.netcdf_promote = True
+    cube1 = iris.load_cube(iris.sample_data_path('ostia_monthly.nc'))
+    # Load the data
+    plot_cube1 = cube1[-1, :, :]
+    # Use only one time slice for data analysis, making a 2D cube
+    plot_line = plot_cube1.collapsed('latitude', iris.analysis.MEAN)
+    # Average over latitude to make a 1D cube
+    region = cube1[0, :, :]
```

Comment:
This is a different time selection that the one you used to get the data for the 1d line plot, it may be nicer if you used the same data.